src/utils/routes.js

```diff
@@ -31,6 +31,11 @@ export default [
     path: `${routes.addAccount.path}/`,
     params: 'dialog',
     name: 'add-account',
+  }, {
+    regex: /\/accounts(?:\/[^/]*)?$/,
+    path: `${routes.accounts.path}/`,
+    params: 'dialog',
+    name: 'accounts',
```

Comment:
It should be possible to remove this whole file because the only left modal (proxyLogin) doesn't have it's own path.

Reply:
done